### DISCUSSION TITLE
MultiSensor 6 and Target Endpoint

### DISCUSSION THREAD

**Discussion by dirandad** [zwave-js/zwave-js-ui#2142] [Q&A]
Hello,

I have an Aeon MultiSensor 6 that I want to switch on/off a Fibaro FGS223 (2 switchs so multiendpoint).
When I configure Group association in the MultiSensor I cannot select the target endpoint so that only first switch of a FGS can be switched.
Is it a limitation of the Multisensor or of zwave-js ?

Thank you.

Adrien

**Comment by robertsLando** [comment]
If you cannot select the target endpoint it means that association doesn't support it.
